common/common-lua.cc

```diff
@@ -254,6 +254,13 @@ void setupCommonLua(bool client,
     c_lua.writeFunction("setNumWebHookConnsPerThread", [](unsigned int num_conns) { });
   }
 
+  if (!multi_lua) {
```

Comment:
Somehow I feel it might be a good idea at some point to move all this `!multi_lua` stuff into some separate method/function/whatever that will initialize these instead of the other. Would be cleaner and easier to read.

Reply:
you have quite a lot of those `if (!multi_lua) {` branches here. you could have `setupCommonBase´, `setupCommonMultiLua` and `setupCommonLua` instead.